main.py

```diff
@@ -2,6 +2,26 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    drama = Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Clooney")
+    Actor.objects.create(first_name="Keanu", last_name="Reeves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+    drama.name = "Drama"
+    drama.save()
+    Actor.objects.filter(first_name="George",
+                         last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu",
+                         last_name="Reaves").update(first_name="Keanu",
+                                                    last_name="Reeves")
```

Comment:
The filter condition uses 'Kianu' and 'Reaves' instead of 'Keanu' and 'Reeves'. This will not match any records unless there is an intentional misspelling in your database.